generator.yaml

```diff
@@ -29,19 +29,61 @@ resources:
         404:
           code: ValidationException
           message_prefix: Could not find model
+      terminal_codes:
+        - ResourceLimitExceeded
+        - ResourceNotFound
```

Comment:
ResourceNotFound and ResourceInUse arent listed in Create/Describe/Update/Delete Endpoint, Model, and EndpointConfig errors. (probably because it throws ValidationExceptions for these cases as well)

should we only list the errors each of these resources can throw? 

to confirm, have you checked all the APIs per resource to make sure they don't have other error types

Reply:
APIs are not consistent in how the exceptions are thrown. I've mostly seen validationExceptions even when it the API has other Exceptions for that situation. It is hard to tell at this time which exceptions are thrown and not. So, we decided to do keep the exceptions same across board for now and iterate.